src/Sulu/Bundle/AdminBundle/Resources/js/components/Grid/tests/__snapshots__/Grid.test.js.snap

```diff
@@ -1,122 +1,9 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`Render a Grid with Items having spaces between them 1`] = `
-<div
-  class="grid"
->
-  <div
-    class="item colSpan colSpan-4 space-before-0 space-after-8"
-  />
-  <div
-    class="item colSpan colSpan-2 space-before-10 space-after-0"
-  />
-</div>
-`;
+exports[`Render a Grid with Items having spaces between them 1`] = `undefined`;
```

Comment:
Looks like missing import for react testing library. Still `enzyme` being used. :)

Reply:
@b3h3m0th looks like you forget to update the snapshots after adding the correct import here: https://github.com/sulu/sulu/pull/6746/commits/89c1f33bf289b1995675198c7541ecd3e52c4648